topics/epigenetics/metadata.yaml

```diff
@@ -54,6 +54,26 @@ material:
     contributors:
       - joachimwolff
       - dpryan79
+  -
+    title: "Hi-C analysis of mouse ESCs using HiCExplorer"
```

Comment:
Replace title with `Hi-C analysis of Drosophila melanogaster cells using HiCExplorer`